lib/apress/selenium_eti/spec/company_site/eti/product_creation_spec.rb

```diff
@@ -65,36 +65,37 @@
         @product = {
           name: Faker::Name.title,
           rubric: CONFIG['eti']['rubric'],
-          portal_traits: {
-            gost: CONFIG['eti']['portal_traits']['gost'],
-            condition: CONFIG['eti']['portal_traits']['condition'],
-            manufacturer_country: CONFIG['eti']['portal_traits']['manufacturer_country']
-          },
           exists: CONFIG['eti']['exists']['in stock'],
           short_description: CONFIG['product_creation']['short_description']['valid'],
           description: 'description',
           price_from_to: {from: Faker::Number.number(3), to: Faker::Number.number(4)},
           wholesale_price: {wholesale_price: Faker::Number.number(2), wholesale_number: Faker::Number.number(2)}
         }
 
-        @cs_eti_page.create_and_set_product_fields(@product)
+        @portal_traits = {
+            size: CONFIG['eti']['portal_traits']['size'],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.